docs/windows.md

```diff
@@ -145,3 +145,41 @@ Now that everything is setup, let's build a [simple _hello world_ image](https:/
     ```
 
 > **NOTE:** After pushing to the registry, you can use your image with any other clients to spin off containers, e.g. `docker run`, `ctr run`, `nerdctl run`, etc.
+
+## Running `buildctl` from a Non-Admin Terminal
+
+The default case for running `buildctl` is from an admin (elevated) terminal.
+If you attempt running in a non-admin terminal, you will get an `Access Denied`
+error, on the named pipe:
+
+```
+connection error: desc = "transport: Error while dialing: open \\\\.\\pipe\\buildkitd: Access is denied."
+```
+
+However, it is possible to run it in a non-admin terminal by providing
+the group name(s) of the users executing the command.
+
+You can find the group names that the current user belongs too by running:
+
+```cmd
+whoami /groups
+```
+
+You can also create a group and add the user on it by running the following in
+an admin terminal:
+
+```powershell
+# you can use $env:USERNAME for PowerShell or
+# %USERNAME% for CMD terminal
+net localgroup buildkit-users <username> /add
```

Comment:
```powershell
PS>net localgroup "buildkit-users" "${env:USERNAME}" /add
The specified local group does not exist.
```
```powershell
PS>net localgroup "buildkit-users ${env:USERNAME}" /add
The command completed successfully.
```

Reply:
that must be a PowerShell nuance, just without quotes should work: `net local group buildkit-users $env:USERNAME` on PS and `net local group buildkit-users %USERNAME%` on CMD. Or just using the username you know directly without using the envs.

Reply:
Oh, and remember, you have to run the `net localgroup buildkit-user /add` first (creating the group) before adding the user to the group. If you ran your command first: `>net localgroup "buildkit-users ${env:USERNAME}" /add`, then it means you were creating a group named `buildkit-users <username>`. That could have been your issue.